example/package.json

```diff
@@ -2,7 +2,10 @@
   "name": "oscal-viewer",
   "homepage": ".",
   "version": "0.1.0",
-  "private": true,
```

Comment:
Not sure about this change. I thought it needed to be removed in order to publish - can someone clarify what this line was doing or if it's still necessary?